src/component/signUp/SignUp.scss

```diff
@@ -2,42 +2,49 @@ body {
   font-family: 'Roboto', sans-serif;
   font-weight: 300;  
   background-color: #C5E9FF;
-  .signUp {    
+  
+  .signUp {
     width: 80vw;
```

Comment:
in width, in my opinion, you can use rem

Reply:
recently i wrote same sentence for using 'vh' height attribute. I have same question for use 'vw'